bucket/element.json

```diff
@@ -12,13 +12,13 @@
     "extract_dir": "lib\\net45",
     "bin": [
         [
```

Comment:
```suggestion
        "Element.exe",
        [
```

Reply:
Why do we need two different shims for the same executable?

Reply:
Becuase in documentaiton they use `element-desktop` ...
![image](https://user-images.githubusercontent.com/13260377/88651648-3044ae00-d0ca-11ea-9a76-b2f80e565eff.png)

Reply:
Yeah I know, but why can't we just have `element-desktop` then?

Reply:
Because it is its name? Original executable.

Before it was not created before its stupid `(riot) in name. Now they removed and it could be added.